### PR TITLE
Include zpages data in flare

### PR DESCRIPTION
**Pull request by mackjmr**
### What does this PR do?

This PR adds the zpages data from endpoints `/debug/{servicez, pipelinez, extensionz, featurez, tracez}`. It also removes the auto crawl logic.

### Motivation

OTEL-251

### Describe how to test/QA your changes

Run otel-agent (`inv otel-agent.build && ./bin/otel-agent/otel-agent run --config collector.yaml`) + agent (`inv agent.build && ./bin/agent/agent run -c bin/agent/dist/datadog.yaml`) at the same time.

Generate a flare: `sudo datadog-agent flare`. If you see error `invalid session token`, you can return `nil` in this [function](https://github.com/DataDog/datadog-agent/blob/main/pkg/api/util/util.go#L90) for local testing purposes.

Ensure that the flare contains the zpages data:

And that each page is visible in html format via an html viewer such as https://codebeautify.org/htmlviewer.

### CODE REVIEW

File: comp/otelcol/ddflareextension/impl/extension.go [hunk 1]
```diff
@@ -101,8 +113,7 @@ func (ext *ddExtension) Start(_ context.Context, host component.Host) error {
 		} else {
```
[the remark this answers is not in the document]
Comment (mackjmr) [reply comp/otelcol/ddflareextension/impl/extension.go]: It's in a range, so added a `continue` statement instead of return.

**Review by truthbk — APPROVED** [review]

**Comment by jackgopack4** [comment]
Is there a specific config.yaml you used to test this? I am having trouble finding one to get the otel agent to build, even the "default" one in comp/otelcol/collector-contrib/impl/manifest.yaml

**Comment by mackjmr** [comment]
> Is there a specific config.yaml you used to test this? I am having trouble finding one to get the otel agent to build, even the "default" one in comp/otelcol/collector-contrib/impl/manifest.yaml

discussed offline

**Review by jackgopack4 — APPROVED** [review]
Looks good to me, thanks for explaining a few things offline about this functionality.

**Pull request merged**